nix/modules/sequencer/sequencer-opts.nix

```diff
@@ -79,7 +79,7 @@ let
 
       safe_min_quorum = mkOption {
         type = types.int;
-        default = 5;
+        default = 1;
```

Comment:
A reminder for near future - we might want to change this parameter to be the percentage of reporters that form a quorum.